src/components/pages/RoleSignup/Applications/Mentor.js

```diff
@@ -72,444 +104,413 @@ const Mentor = ({ dispatch, error, successPage }) => {
+                    },
+                  ]}
+                >
+                  <Input placeholder="Current company" />
+                </Form.Item>
+              </Col>
+              {/*Position*/}
+              <Col md={20} xs={24}>
+                <Form.Item
+                  label="Current position"
+                  type="text"
+                  name="current_position"
+                  value={formValues.current_position}
+                  onChange={handleChange}
+                  rules={[
+                    {
+                      required: true,
+                      message: 'Current position is required.',
+                    },
+                  ]}
+                >
+                  <Input placeholder="Current position" />
+                </Form.Item>
+              </Col>
+            </Row>
 
-              <Row style={{ padding: '3% 0 3% 3%' }}>
-                <Col className="test" md={22} xs={24}>
-                  <Form.Item
-                    className="mentorshipArea"
-                    label="What areas are you wanting to provide mentorship in?"
-                    tooltip={{
-                      title: 'What development role have you trained for?',
-                      icon: <InfoCircleOutlined />,
+            {/*CheckBoxes*/}
+            <Row>
+              <Col className="mentorshipArea" md={20} xs={24}>
+                <Form.Item
+                  size="medium"
+                  className="form-group"
+                  label="What areas are you wanting to provide mentorship in?"
+                  tooltip={{
+                    title: 'What development role have you trained for?',
+                    icon: <InfoCircleOutlined />,
+                  }}
+                  name="tech_stack"
+                  rules={[
+                    {
+                      required: true,
+                      message: 'Tech stack is required.',
+                    },
+                  ]}
+                >
+                  <Checkbox.Group
+                    style={{
+                      display: 'flex',
+                      flexFlow: 'column',
+                      width: 'auto',
+                      gap: '.5rem',
+                      paddingLeft: '1rem',
                     }}
-                    name="tech_stack"
-                    rules={[
-                      {
-                        required: true,
-                        message: 'Tech stack is required.',
-                      },
-                    ]}
                   >
-                    <Checkbox.Group
-                      style={{
-                        display: 'flex',
-                        justifyContent: 'space-evenly',
-                        flexFlow: 'column',
-                        width: 350,
-                        margin: '0rem 1rem 1rem 1.5rem',
-                      }}
-                    >
+                    {mentorshipArray.map(checkbox => (
```

Comment:
As @MelodyLayne noted with a separate example in another section, good job replacing hard-coded options with a concise programmatic solution!